storm-core/src/clj/backtype/storm/daemon/logviewer.clj

```diff
@@ -476,6 +485,405 @@ Note that if anything goes wrong, this will throw an Error and exit."
       (-> (resp/response "Page not found")
           (resp/status 404)))))
 
+(def grep-max-search-size 1024)
+(def grep-buf-size 2048)
+(def grep-context-size 128)
+
+(defn logviewer-port
```

Comment:
We have it have a function mostly for mocking.
logviewer_test.clj
let[expected-port 8888]
    (stubbing [local-hostname expected-host
               logviewer/logviewer-port expected-port]
     .................